dccrg.hpp

```diff
@@ -3180,6 +3180,98 @@ template <
 		return ret_val;
 	}
 
+	/*!
+	Transfers unrefined cell data between processes.
+
+	Must be called by all processes and not before start_refining()
+	has been called.
+
+	The next function to be called after this one (from those that
+	must be called by all processes) must be either this again or
+	finish_refining().
+
+	\see
+	finish_refining()
+	*/
+	Dccrg<
+		Cell_Data,
+		Geometry,
+		std::tuple<Additional_Cell_Items...>,
+		std::tuple<Additional_Neighbor_Items...>
+	>& continue_refining()
+	{
+		// No check whether start_refining() has been called...
```

Comment:
please change these to // TODO: check...

Reply:
I can just implement the checks if desired.